config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'experiences/index'
```

Comment:
We're gonna use ressources for the routes, can you get rid of those two lines please? ;) @sashakaverina

Reply:
fixed! @yannklein